### PR TITLE
Documentation Fixes for PSC Setup

### PR DESCRIPTION
**Pull request by teor2345**
This branch fixes some typos and mistakes in the PSC documentation.
It also points to the PrivCount documentation where appropriate.

### CODE REVIEW

**Comment by AkshayaMani** [comment]
Hello Tim,

Thank you. I will look into it. PSC does support Safe Cookie Authentication
now. I have fixed it.

Thanks,
Akshaya


On Feb 8, 2018 8:42 PM, "teor" <notifications@github.com> wrote:

This branch fixes some typos and mistakes in the PSC documentation.
It also points to the PrivCount documentation where appropriate.
------------------------------
You can view, comment on, or merge this pull request online at:

  https://github.com/AkshayaMani/PSC/pull/1
Commit Summary

   - Fix PSC update path in INSTALL.markdown
   - Use goenv for the go environment name in README.md
   - Use goenv for the go environment name in INSTALL.md
   - Use nv to activate the goenv in DEPLOY.markdown
   - PSC does not support Cookie Authentication
   - Remove duplicate "Tor Dependencies" line
   - Point to the PrivCount install instructions
   - Point to the PrivCount instructions in DEPLOY.markdown
   - Avoid use of sudo when creating CA files

File Changes

   - *M* CAInstructions.markdown
   <https://github.com/AkshayaMani/PSC/pull/1/files#diff-0> (26)
   - *M* DEPLOY.markdown
   <https://github.com/AkshayaMani/PSC/pull/1/files#diff-1> (25)
   - *M* INSTALL.markdown
   <https://github.com/AkshayaMani/PSC/pull/1/files#diff-2> (12)
   - *M* README.md <https://github.com/AkshayaMani/PSC/pull/1/files#diff-3>
   (2)

Patch Links:

   - https://github.com/AkshayaMani/PSC/pull/1.patch
   - https://github.com/AkshayaMani/PSC/pull/1.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/AkshayaMani/PSC/pull/1>, or mute the thread

**Comment by teor2345** [comment]
I have force-pushed an updated branch which removes that commit, and squashes some of the other commits.

File: DEPLOY.markdown [hunk 1]
```diff
@@ -65,18 +66,12 @@ Be careful while collecting and releasing PSC results: the configured epsilon an
 
 PSC is not designed for automated collection and results release: a long enough series of results can identify the activity of a single user.
 
-After configuring parameters, run Gen_Config.go:
```
Comment (AkshayaMani) [inline DEPLOY.markdown]: The Tally Server needs to generate the configuration parameters before it can be run.

File: INSTALL.markdown [hunk 2]
```diff
@@ -22,11 +22,11 @@ Restart your terminal.
 
 ### Create an environment <goenvironment_name> with go 1.7 or later
```
Comment (AkshayaMani) [inline INSTALL.markdown]: ### Create an environment goenv with go 1.7 or later

File: DEPLOY.markdown [hunk 3]
```diff
@@ -57,6 +57,7 @@ Set the PSC parameters in PSC/TS/config/Gen_Config.go:
 And generate configuration:
 
 ```
+    nv on goenv
     cd PSC/TS/config/
```
Comment (AkshayaMani) [inline DEPLOY.markdown]: cd $GOPATH/src/PSC/TS/config/

File: DEPLOY.markdown [hunk 4]
```diff
@@ -65,18 +66,12 @@ Be careful while collecting and releasing PSC results: the configured epsilon an
...
-    go run Gen_Config.go
-```
-
 #### PSC      
 
 Then run PSC in Tally Server mode:
 
 ```
+    nv on goenv
     cd PSC/TS/
```
Comment (AkshayaMani) [inline DEPLOY.markdown]: cd $GOPATH/src/PSC/TS/

File: DEPLOY.markdown [hunk 5]
```diff
@@ -107,6 +102,7 @@ Set the TS information in PSC/CP/ts.info:
 Run PSC in Computation Party mode:
 
 ```
+    nv on goenv
     cd PSC/CP/
```
Comment (AkshayaMani) [inline DEPLOY.markdown]: cd $GOPATH/src/PSC/CP/

**Comment by teor2345** [comment]
Similar changes have recently been merged to master.

**Comment by teor2345** [comment]
@AkshayaMani I don't mind you rewriting these pull requests. But if you work on other open-source projects, it's best to leave authors intact on commit messages.

**Pull request closed without merging**